src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java

```diff
@@ -75,11 +78,18 @@
             optional = true)
     protected File truePositivesAndFalsePositivesVcf = null;
 
+    @Argument(doc = "A vcf to write filtered true negatives and false negatives",
+            fullName = FILTERED_TRUE_NEGATIVES_AND_FALSE_NEGATIVES_LONG_NAME,
+            shortName = FILTERED_TRUE_NEGATIVES_AND_FALSE_NEGATIVES_SHORT_NAME,
+            optional = true)
+    protected File filteredTrueNegativesAndFalseNegativesVcf = null;
+
     // we count true positives, false positives, false negatives for snps and indels
     private final EnumMap<ConcordanceState, MutableLong> snpCounts = new EnumMap<>(ConcordanceState.class);
     private final EnumMap<ConcordanceState, MutableLong> indelCounts = new EnumMap<>(ConcordanceState.class);
     private VariantContextWriter truePositivesAndFalseNegativesVcfWriter;
     private VariantContextWriter truePositivesAndFalsePositivesVcfWriter;
+    private VariantContextWriter filteredTrueNegativesAndFalseNegativesVcfWriter;
```

Comment:
I ask because on line 137 we just `return`; do we write elsewhere?

```java
        } else if (concordanceState == ConcordanceState.FILTERED_TRUE_NEGATIVE) {
            return;
```